.gitignore

```diff
@@ -4,3 +4,15 @@ build
 dist
 happy.egg-info
 demos/.ipynb_checkpoints
+
```

Comment:
Hmm, I was hoping to ignore all of these files below at the directory level via the "build" ignore rule (i.e. they should be omitted by being under the build/ directory). Did CMake create a different directory for these files?

Reply:
oh that might have been my doing